src/error/metadata/suggestion.rs

```diff
@@ -110,7 +111,11 @@ impl Display for Suggestion {
                     Yellow.normal().paint("`rover config auth`")
                 )
             }
-            Suggestion::Adhoc(msg) => msg.to_string()
+            Suggestion::DisabledIntrospection => {
+                "Introspection might be disabled for this endpoint.".to_string()
```

Comment:
This suggestion doesn't seem to quite be what you laid out in the PR description, I wonder if we're missing a commit here?

Reply:
yea, let me remove this. This shouldn't be here.